css/contacts.scss

```diff
@@ -38,12 +38,12 @@ $grid-input-height-with-margin: #{$grid-height-unit - $grid-input-margin * 2};
 }
 
 
-@import 'settings-section';
-@import 'settings/settings-addressbook';
-@import 'settings/settings-addressbook-shares';
-@import 'settings/settings-addressbook-sharee';
+@import 'SettingsSection';
+@import 'Settings/SettingsSddressbook';
```

Comment:
Still this typo ;)

Reply:
You mean this one? 
```
@import 'SettingsSection';
@import 'Settings/SettingsSddressbook';
@import 'Settings/SettingsAddressbookShares';
@import 'Settings/SettingsAddressbookSharee';```

Reply:
Or this 
```
"Sharee"
```

Reply:
No no, the Sharee is fine! Only the A in Addressbook :)